src/project/projectmodule.cpp

```diff
@@ -137,6 +138,9 @@ void ProjectModule::registerUiTypes()
 {
     qmlRegisterType<ExportDialogModel>("MuseScore.Project", 1, 0, "ExportDialogModel");
 
+    qmlRegisterType<ScoresPageModel>("MuseScore.Project", 1, 0, "ScoresPageModel");
+    qmlRegisterUncreatableType<AbstractScoresModel>("MuseScore.Project", 1, 0, "AbstractScoresModel",
```

Comment:
Why do you need to register this abstract model?

Reply:
I was planning to specify it as the type of the `model` property in QML. I initially forgot to actually do that, but now it's done :)